src/main/frontend/src/components/FoodChainForm.js

```diff
@@ -84,8 +84,8 @@ const FoodChainForm = props => {
       name="img_url"
       placeholder="https://www.readersdigest.ca/wp-content/uploads/2018/01/banned-mcdonalds.jpg"
       value={newFoodChain.imgUrl}
-      onChange={handleInputChange}
-      ></input>
+      onChange={handleInputChange}>
+      </input>
```

Comment:
This is not the fix I meant, this input is within the label therefore the whole entire input should be indented within the label.